worlds/factorio/Options.py

```diff
@@ -146,8 +146,8 @@ class TechTreeLayout(Choice):
 
 class TechTreeInformation(Choice):
     """How much information should be displayed in the tech tree.
-    None: No indication what a research unlocks
-    Advancement: Indicators which researches unlock items that are considered logical advancements
+    None: No indication of what a research unlocks
+    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for
```

Comment:
```suggestion
    Advancement: Indicates if a research unlocks an item that is considered logical advancement, but not who it is for
```
Small suggestion for readability

Reply:
Ah, by applying the other one I have made yours outdated, dang it.

Reply:
Handled in 7bc046f